source/_docs/guides/partial-composer.md

```diff
@@ -0,0 +1,231 @@
+
+## Before You Begin
+- Read [Composer Fundamentals and Workflows](/docs/composer/)
+- Install [Composer](https://getcomposer.org/doc/00-intro.md#installation-linux-unix-osx){.external} and [Git](https://git-scm.com/downloads){.external} locally
+- Create a WordPress or Drupal 7 site on Pantheon
+
+  {% include("content/notes/partial-composer-adoption-warning.html") %}
+
+- Set the site's connection mode to Git within the Site Dashboard or via [Terminus](/docs/terminus):
+
+  ```bash
+  terminus connection:set <site>.<env> git
+  ```
+
+- Create a local clone of your site code, and navigate to it in your terminal
+
+
+## Initialize and Configure Composer
+Use the `init` command to create a `composer.json` file that includes the appropriate package repository, then configure installation paths for dependencies like plugins and modules:
+
+  <!-- Nav tabs -->
+  <ul class="nav nav-tabs" role="tablist">
+    <!-- Active tab -->
+    <li id="wp-init-id" role="presentation" class="active"><a href="#wp-init" aria-controls="wp-init" role="tab" data-toggle="tab">WordPress</a></li>
+
+    <!-- 2nd Tab Nav -->
+    <li id="d7-init-id" role="presentation"><a href="#d7-init" aria-controls="d7-init" role="tab" data-toggle="tab">Drupal 7</a></li>
+  </ul>
+
+  <!-- Tab panes -->
+  <div class="tab-content">
+    <!-- Active pane content -->
+    <div role="tabpanel" class="tab-pane active" id="wp-init" markdown="1">
+    1. Initialize composer to create a `composer.json` file with the WordPress package repository:
+
+      ```command
+      composer init --repository=https://wpackagist.org --no-interaction
+      ```
+
+    2. Edit the `composer.json` to add extra configuration that specifies installation paths for WordPress plugins and themes:
+
+      ```json
+      {
+          "repositories": [
+              {
+                  "type": "composer",
+                  "url": "https://wpackagist.org"
+              }
+          ],
+          "require": {},
+          "extra": {
+            "installer-paths": {
+              "wp-content/plugins/{$name}/": ["type:wordpress-plugin"],
+              "wp-content/themes/{$name}/": ["type:wordpress-theme"]
+            }
+          }
+      }
+      ```
+
+    3. Commit the `composer.json` file to version control with Git:
+
+      ```command
+      git add composer.json
+      ```
+
+      ```command
+      git commit -m "Create composer.json with WP repo and install paths"
+      ```
+
+    4. Push your new file to Pantheon:
+
+      ```command
+      git push origin master
+      ```
+    </div>
+
+    <!-- 2nd pane content -->
+    <div role="tabpanel" class="tab-pane" id="d7-init" markdown="1">
+    1. Initialize composer to create a `composer.json` file with the Drupal 7 package repository:
+
+      ```command
+      composer init --repository=https://packages.drupal.org/7 --no-interaction -s=alpha
```

Comment:
Is stability: alpha the best recommendation here? Maybe we should say `-s=stable`. This is the default, but putting it in would make it clear how folks should alter this if they want to use unstable modules.

Reply:
Ah, good catch.  I’ll fix this up. The guide goes through requiring the PAPC module in the following section so that’s why it’s alpha here. Planning to switch that demo to a package that doesn’t depend on drupal to avoid overwriting core, once I do that I’m not sure we’ll need to specify stability (unless we have specific suggestions, otherwise we can link to composer docs).